oat-sa/extension-tao-itemqti
# Fix/tr 3076/support bidirectonal feedback instruction

**Related to:** [TR-3076](https://oat-sa.atlassian.net/browse/TR-3076)

**Description:**
Support bidirectional feedback instruction on items. Check User language on item language with different directions.


**Changes:**

- Add CSS rule for the case User LTR.

**How to check:**

- Create a choice interaction and set the min and max choices to show the item instruction.
- Duplicate the item, so one is in Arabic and the other in English.
- Create an Arabic User and English user (or use the Guest User), and check the 4 case scenario:
User RTL + Item RTL / User RTL + Item LTR / User LTR + Item LTR / User LTR + Item RTL
Check that the item instruction is always is the same direction of the User language direction, even the item has other language direction.
![image](https://user-images.githubusercontent.com/11692751/151763678-b10dbcbc-d0bb-4b0f-9ba0-07ad951535ac.png)

- TAE and Composer in the Environment label of the ticket
